develop/parachains/testing/mock-runtime.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Mock Runtime for Pallet Testing
+description: Learn how to create a mock environment to test complete pallet functionalities in the Polkadot SDK, ensuring integration between pallets and system components.
+---
+
+# Mock Runtime
+
+## Introduction
+
+Testing is essential in Polkadot SDK development to ensure your blockchain operates as intended and effectively handles various potential scenarios.
+
+This guide will walk you through setting up an environment to test pallets within the runtime. Doing so lets you evaluate how different pallets, their configurations, and system components interact, ensuring your blockchain functions reliably under real-world conditions.
+
+## Configuring a Mock Runtime
+
+### Testing Module
+
+The mock runtime includes all the necessary pallets and configurations needed for testing. To simplify the process, you can create a module that integrates all components, making it easier to assess how pallets and system elements interact.
```

Comment:
> To simplify the process, you can create a module that integrates all components, making it easier to assess how pallets and system elements interact.

Having a mock runtime is mandatory to do any testing, not the make thigns "easier".

Reply:
fixed!